vocode/streaming/agent/state_agent.py

```diff
@@ -1583,6 +1625,8 @@ async def call_ai(self, prompt, tool=None, stop=None, stream_output=True):
                 and send_final_message
                 and buffer.strip() != '"}'
                 and buffer.strip() != "}"
+                and buffer.strip() != '"}\n```'
+                # https://app.opencall.ai/console/calls/WzEsICJwdWJsaWMiLCAiY2FsbF9kZXRhaWxzIiwgODU4OTZd
```

Comment:
Keep getting a } in the transcripts. Just need to remove it properly before adding as a `produce_interruptible_agent_response_event_nonblocking`

Reply:
Not currently being removed properly. Even with this change.